godel_robots/abb/abb_file_suite/src/ftp_upload.cpp

```diff
@@ -0,0 +1,139 @@
+#include "ftp_upload.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <curl/curl.h>
+
+/* parse headers for Content-Length */
+static size_t getcontentlengthfunc(void *ptr, size_t size, size_t nmemb, void *stream)
+{
+  int r;
```

Comment:
It'd be nice to rename this variable to something slightly more descriptive of its role.